secure_tunneling/include/aws/iotsecuretunneling/SecureTunnel.h

```diff
@@ -0,0 +1,100 @@
+#pragma once
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+#include <aws/crt/io/Bootstrap.h>
+#include <aws/crt/io/SocketOptions.h>
+#include <aws/iotdevice/secure_tunneling.h>
+#include <aws/iotsecuretunneling/Exports.h>
+
+int s_SecureTunnelingCallbackTest(Aws::Crt::Allocator *allocator, void *ctx);
+
+namespace Aws
+{
+    namespace Iotsecuretunneling
+    {
+        class AWS_IOTSECURETUNNELING_API DeviceApiHandle final
```

Comment:
This doesn't beloing in Iotsecuretunneling, it belongs in a top level Iot namespace most likely. This will apply to DD as well.

Reply:
DD and ST are built as separate libraries. What happen when the customer uses both libraries. If I put this in the Iot namespace, I would need to name `DeviceApiHandle` to `SecureTunnelingDeviceApiHandle`, right?

Reply:
Do we need a top level iotdevice folder that can declare the DeviceAPIHandle so that DD and ST can share that?

Reply:
> Do we need a top level iotdevice folder that can declare the DeviceAPIHandle so that DD and ST can share that?

That seems like the right move. This is where non-generated code would go that is common to IoT

Reply:
The work is done in this PR:
https://github.com/aws/aws-iot-device-sdk-cpp-v2/pull/184